### PR TITLE
New privacy policy

### PR DESCRIPTION
**Pull request by krzemin**
Generated with the help of ChatGPT.

### CODE REVIEW

File: _pages/privacypolicy.md [hunk 1]
```diff
@@ -5,12 +5,60 @@ include_in_header: true
 ---
 
 **Last updated**  
-October 24, 2020
+June 26, 2023
```
Comment (krzemin) [inline _pages/privacypolicy.md]: to be updated before merging

File: _pages/privacypolicy.md [hunk 2]
```diff
@@ -5,12 +5,56 @@ include_in_header: true
...
+
+The information we collect is used solely for the purpose of improving our app and enhancing the user experience. Specifically, we may use the collected information for the following purposes:
+
+- App Improvement: We analyze the app logs to identify and fix bugs, errors, and other technical issues within our app. This helps us provide a more stable and reliable app experience for our users.
+
+- User Experience Enhancement: The collected information allows us to understand how users interact with our app, including the features they use most frequently or encounter difficulties with. This enables us to make informed decisions to enhance the user interface and overall user experience.
+
+## 3. Personal Data
+
+We want to emphasize that our app does not collect any private or personal data, including the following:
```
Comment (NoemiRozpara) [inline _pages/privacypolicy.md]: I'd emphasize it more: 
```suggestion
We want to emphasize that our app does NOT collect any private or personal data, like:
```

File: _pages/privacypolicy.md [hunk 3]
```diff
@@ -5,12 +5,56 @@ include_in_header: true
...
+- MPD server credentials (host names, ports, passwords)
+- Contents of your MPD queue and library
+- Titles of songs you listen to
+- Local playback stream URLs
+
+We respect your privacy and are committed to ensuring that no private or personal data is collected through our app.
+
+## 4. Data Sharing and Disclosure
+
+We do not sell, rent, or trade any of the information collected through our app, including the app logs. We may share the collected information in an aggregated and anonymized form with third parties for analytical and statistical purposes.
```
Comment (NoemiRozpara) [inline _pages/privacypolicy.md]:
```suggestion
We do not sell, rent, or trade any of the information collected through our app, including the app logs. We store  the collected information in an aggregated and anonymized form in the third party software for the internal use only.
```

File: _pages/privacypolicy.md [hunk 4]
```diff
@@ -5,12 +5,56 @@ include_in_header: true
...
+
+## 5. Data Retention
+
+We retain the collected information, including the app logs, for as long as necessary to fulfill the purposes outlined in this Privacy Policy. Once the information is no longer required, we will securely delete or anonymize it.
```
Comment (NoemiRozpara) [inline _pages/privacypolicy.md]:
```suggestion
We retain the collected information, including the app logs, for as long as necessary to fulfill the purposes outlined in this Privacy Policy. Once the information is no longer required, we will securely delete it.
```
The data is already anonymized

File: _pages/privacypolicy.md (hunk 1 above)
Comment (krzemin) [reply _pages/privacypolicy.md]: ✅

**Pull request merged**